swap_meet/clothing.py

```diff
@@ -1,2 +1,10 @@
-class Clothing:
-    pass
+from swap_meet.item import Item
+
+class Clothing(Item):
+    def __init__(self, condition=0):
+        super().__init__(category="Clothing", condition=condition, id="The finest clothing you could wear.")
+    
+    def __str__(self):
+        return str(self.id)
```

Comment:
You don't actually need to override this, since this is the same as `Item`'s method.